crates/forge/src/run_tests.rs

```diff
@@ -1,3 +1,4 @@
+pub mod helpers;
```

Comment:
Please no helpers module. Place this somewhere where it is relevant, because this will eventually contain random, unrelated code